spec/factories/users.rb

```diff
@@ -0,0 +1,24 @@
+FactoryBot.define do
+  factory :user do
+    utorid { "utorid#{Faker::Number.number(2)}" }
+    
+    trait :admin do
+      role { 0 }
```

Comment:
I think we should do symbols instead of integers. It's hard to read otherwise and we need to look back. Also, I don't like magic constants.